cloud/whitelist.go

```diff
@@ -53,8 +54,9 @@ func (w *WhiteList) Check(existing, new string) error {
 // CurrentWhiteList returns current clouds whitelist supported by Juju.
 func CurrentWhiteList() *WhiteList {
 	return &WhiteList{map[string]set.Strings{
-		lxdnames.ProviderType: set.NewStrings(lxdnames.ProviderType, "maas", "openstack"),
-		"maas":                set.NewStrings("maas", "openstack"),
-		"openstack":           set.NewStrings("openstack"),
+		k8sconstants.CAASProviderType: set.NewStrings(lxdnames.ProviderType, "maas", "openstack"),
```

Comment:
so this allows us to add lxd cloud without --force?
do we wanna enable `lxdnames.ProviderType` now?

Reply:
yeah, --force no longer needed as this PR allows remote lxd clouds to be added successfully